_docs/setup/consul/faq.md

```diff
@@ -3,18 +3,17 @@ title: FAQ
 overview: Frequently asked questions, current limitations and troubleshooting tips.
 
 order: 100
-draft: true
 
 layout: docs
 type: markdown
 ---
 
 {% include home.html %}
 
-* _My application isn't working, where can I start by debugging this?_
+* _My application isn't working, where can I troubleshoot this?_
 
   Please ensure all required containers are running: etcd, istio-apiserver, consul, registrator, pilot.  If one of them is not running, you may find the {containerID} using `docker ps -a` and then use `docker logs {containerID}` to read the logs.   
   
 * _How do I unset the context changed by istioctl at the end?_
```

Comment:
Do we even need this piece? We are trying to actively avoid the kubectl stuff in Consul.

Reply:
yes we do, at least for the machines who are intended to use kubectl for other stuff.  I run into this myself on my laptop.